app/models/target.rb

```diff
@@ -25,9 +25,16 @@ class Target < ApplicationRecord
   validates :radius, presence: true, numericality: { greater_than: 0 }
   validates :lat, :lon, presence: true, numericality: true
 
+  scope :exclude_current_user, ->(current_user_id) { where.not(user_id: current_user_id) }
```

Comment:
```suggestion
  scope :from_other_users, ->(user_id) { where.not(user_id: user_id) }
```
This scope will return targets from other users. Here is a naming suggestion. Not a blocker.

Reply:
Suggestion taken! Thnks